PrismSampleApp/PrismSampleApp/ViewModels/ViewBViewModel.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Prism.Commands;
+using Prism.Mvvm;
+using Prism.Navigation;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xamarin.Forms;
+
+namespace PrismSampleApp.ViewModels
+{
+    public class ViewBViewModel : BindableBase, INavigationAware
+    {
+        private string _title;
+        public string Title
+        {
+            get
+            {
+                return _title;
+
+            }
+            set
+            {
+                SetProperty(ref _title, value);
+            }
+        }
+
+        IViewAViewModel ViewAViewModel;
+        public ViewBViewModel(INavigationService navigationService)
+        {
+            Title = "View B";
+            _navigationService = navigationService;
+            ViewAViewModel = DependencyService.Get<IViewAViewModel>();
+        }
+        private DelegateCommand _navigateCommand;
+        private readonly INavigationService _navigationService;
+        public DelegateCommand NavigateCommandC =>
+            _navigateCommand ?? (_navigateCommand = new DelegateCommand(ExecuteNavigateCommandC));
+
+        async void ExecuteNavigateCommandC()
+        {
+            await _navigationService.NavigateAsync("ViewC");
+        }
+        private DelegateCommand _navigateCommandMainPage;
+        public DelegateCommand GoHomeCommand =>
```

Comment:
Use the Delegate command as you used it in Xamarin sample.

Reply:
Made changes in the delegate command